{"uuid":"5d44966f-cb54-4eaa-8ce3-385b0fad39d4","historyId":"fc3b2f53386dbab1518acc01d3c091e3","fullName":"Base.IOSBaseTest.tearDown","labels":[{"name":"package","value":"TestCases.IOSBasics"},{"name":"testClass","value":"TestCases.IOSBasics"},{"name":"testMethod","value":"tearDown"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"AppiumTrainingHybridFramework"},{"name":"subSuite","value":"TestCases.IOSBasics"},{"name":"host","value":"PPineda-EDP-100-981"},{"name":"thread","value":"3322@PPineda-EDP-100-981.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"AS_ID","value":"-1"}],"links":[],"name":"tearDown","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Error communicating with the remote browser. It may have died.\nBuild info: version: '4.16.1', revision: '9b4c83354e'\nSystem info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '14.1.2', java.version: '21.0.1'\nDriver info: io.appium.java_client.ios.IOSDriver\nCommand: [ae895792-41e7-4202-8765-3ee106ded1b6, quit []]\nCapabilities {appium:app: /Users/princeanriepineda/Li..., appium:automationName: XCuiTest, appium:databaseEnabled: false, appium:deviceName: iPhone SE (3rd generation), appium:javascriptEnabled: true, appium:locationContextEnabled: false, appium:networkConnectionEnabled: false, appium:platformVersion: 17.2, appium:takesScreenshot: true, appium:udid: D7611651-59B8-4052-84CC-62D..., appium:wdaLaunchTimeout: 60000, appium:webStorageEnabled: false, platformName: IOS}\nSession ID: ae895792-41e7-4202-8765-3ee106ded1b6","trace":"org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.\nBuild info: version: '4.16.1', revision: '9b4c83354e'\nSystem info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '14.1.2', java.version: '21.0.1'\nDriver info: io.appium.java_client.ios.IOSDriver\nCommand: [ae895792-41e7-4202-8765-3ee106ded1b6, quit []]\nCapabilities {appium:app: /Users/princeanriepineda/Li..., appium:automationName: XCuiTest, appium:databaseEnabled: false, appium:deviceName: iPhone SE (3rd generation), appium:javascriptEnabled: true, appium:locationContextEnabled: false, appium:networkConnectionEnabled: false, appium:platformVersion: 17.2, appium:takesScreenshot: true, appium:udid: D7611651-59B8-4052-84CC-62D..., appium:wdaLaunchTimeout: 60000, appium:webStorageEnabled: false, platformName: IOS}\nSession ID: ae895792-41e7-4202-8765-3ee106ded1b6\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:550)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:596)\n\tat io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:307)\n\tat org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:438)\n\tat Base.IOSBaseTest.tearDown(IOSBaseTest.java:50)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)\n\tat org.testng.internal.invokers.TestInvoker.runAfterConfigurations(TestInvoker.java:779)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:755)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\n\tat org.testng.TestRunner.run(TestRunner.java:621)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\n\tat org.testng.TestNG.run(TestNG.java:1082)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\nCaused by: java.io.UncheckedIOException: java.net.ConnectException\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:442)\n\tat io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)\n\tat io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:87)\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:358)\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:188)\n\tat io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:238)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:523)\n\t... 34 more\nCaused by: java.net.ConnectException\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1028)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:280)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$2(PlainHttpConnection.java:238)\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\n\tat java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\nCaused by: java.nio.channels.ClosedChannelException\n\tat java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:202)\n\tat java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:786)\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:874)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)\n\tat java.base/java.security.AccessController.doPrivileged(AccessController.java:571)\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)\n\t... 9 more\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1703661471578,"stop":1703661471578}